### PR TITLE
Tweek `doc/README.md` formatting

### PR DESCRIPTION
**Pull request by gudenau**
Added code tags on several paths and commands.

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -163,4 +163,4 @@ In the `lwjgl3` repository:
 In the `lwjgl3-www` repository:
 
 - `client/routes/customize/BuildConfigurator/lwjgl/nightly.js` (`lwjgl3-www` repository)
-    * Add definition for the binding.
\ No newline at end of file
+    * Add definition for the binding.
```
Comment (TheMrMilchmann) [inline doc/README.md]: I'm curious: What happened in this line?

**Comment by TheMrMilchmann** [comment]
In my opinion, your changes improve the overall readability of the file. However, I think if this is done the environment variables mentioned under "Build Configuration" should also be highlighted. (See [here](https://github.com/LWJGL/lwjgl3/pull/360/files#diff-83250cf6ca107475c7c1bb5b042c2dbdL88).)

File: doc/README.md (hunk 1 above)
Comment (kenzierocks) [reply doc/README.md]: A newline `\n` was added to the end of the file. It is a convention in git to end all files with a newline for better diffs when appending to the end of a file.

**Comment by gudenau** [comment]
Sorry about the new line thing, the editor I used messed up the new line thing.

I'll highlight the build variables and fix the new line when I get home.

**Comment by Spasi** [comment]
Thanks!

**Comment by gudenau** [comment]
No problem!

**Pull request merged**